portfolio/src/main/webapp/script.js

```diff
@@ -17,14 +17,28 @@ function displayText(pageType) {
 }
 
 function getMessage() {
-    fetch("/data").then(response => response.json()).then(commentArray => {
+    fetch("/data").then(response => response.json()).then(responseObj => {
```

Comment:
if you could post a screenshot of how this looks, it would be really helpful!

Reply:
Sure thing! 
<img width="1429" alt="Screen Shot 2020-03-24 at 5 30 07 PM" src="https://user-images.githubusercontent.com/46883117/77489429-55ac6d00-6df5-11ea-8c01-e6ed9606e095.png">